src/mrinufft/operators/base.py

```diff
@@ -130,11 +130,9 @@ def wrapper(self, data, output=None, *args, **kwargs):
         if xp.__name__ == "torch" and is_cuda_array(data):
             # Move them to cupy
             data_ = cp.from_dlpack(data)
-            output_ = cp.from_dlpack(output) if output is not None else None
         elif xp.__name__ == "torch":
             # Move to numpy
             data_ = data.to("cpu").numpy()
-            output_ = output.to("cpu").numpy() if output is not None else None
         else:
```

Comment:
This may break some stuff, so be careful. Some use-cases benefit from passing the output array (for better memory usage)